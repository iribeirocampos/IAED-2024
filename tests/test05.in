p parque1 5 0.30 0.50 15.00
p "Parque 2" 5 0.20 0.40 13.00
e parque1 AA-00-AA 01-01-2024 8:00
e parque1 AA-01-AA 01-01-2024 8:01
e parque1 AA-02-AA 01-01-2024 8:02
e parque1 AA-03-AA 01-01-2024 8:03
e parque1 AA-04-AA 01-01-2024 8:03
e "Parque 2" AA-05-AA 01-01-2024 8:03
e "Parque 2" AA-06-AA 01-01-2024 8:04
e "Parque 2" AA-07-AA 01-01-2024 8:05
e "Parque 2" AA-08-AA 01-01-2024 8:06
e "Parque 2" AA-09-AA 01-01-2024 8:06
p
q
