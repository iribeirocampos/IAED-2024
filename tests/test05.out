parque1 4
parque1 3
parque1 2
parque1 1
parque1 0
Parque 2 4
Parque 2 3
Parque 2 2
Parque 2 1
Parque 2 0
parque1 5 0
Parque 2 5 0
